Advanced Collaborations/procedures/patient/monitoring/HeartRate_generateHeartRate.vail

```diff
@@ -0,0 +1,17 @@
+package patient.monitoring
+PROCEDURE HeartRate.generateHeartRate(heartRate Integer)
+
+var event = {
+    name: Context.preferredUsername(),
+    patientId: Context.username(),
+    heartRate: heartRate,
+    location: {
+        "type": "Point",
+        "coordinates": [
+            -122.272680,
+            37.810370
```

Comment:
I needed to update these coordinates to be the office coordinates (since I'm in the office today).
```suggestion
            -122.0659707,
            37.9067
```